packages/sdk/src/types/Loyalties.ts

```diff
@@ -89,32 +291,139 @@ export interface LoyaltiesCreateCampaignResponse {
 	created_at: string
 	vouchers_generation_status: 'IN_PROGRESS' | 'DONE' | 'FAILED' | 'DRAFT'
 	active: boolean
-	voucher?: LoyaltiesVoucher
 	referral_program?: boolean
 	use_voucher_metadata_schema?: boolean
 	protected?: boolean
 	vouchers_count?: number
 	object: 'campaign'
+	voucher?: CampaignVoucherObjectLoyaltyCard
+	updated_at: string
+	creation_status: 'DONE' | 'IN_PROGRESS' | 'FAILED' | 'DRAFT' | 'MODIFYING'
+	category_id: string
+	categories: CategoryObject
+	loyalty_tiers_expiration: LoyaltyTiersExpirationBalance | LoyaltyTiersExpirationPointsInPeriod
 }
 
-export type LoyaltiesGetCampaignResponse = LoyaltiesCreateCampaignResponse
-
-export interface LoyaltiesUpdateCampaign {
```

Comment:
Interface vs Type - please look at origin comment about the `DistributionsPublicationsCreateParams`